### ISSUE TITLE
Reinstall plugin requires upgrade TablePlus

### ISSUE THREAD

**Issue by vothanhkiet** [TablePlus/TablePlus#1021]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
Any
2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
Build 200
3. The steps to reproduce this issue:
- Plugins > Manage Plugins
- Select any installed plugin on Available Plugins section
- Choose Reinstall

<img width="438" alt="screen shot 2019-03-04 at 19 01 40" src="https://user-images.githubusercontent.com/4302582/53732429-7df6c000-3eb0-11e9-8e7e-26cf6973a99f.png">

**Comment by huyphams** [comment]
Hi @vothanhkiet please remove it before reinstalling the plugin. Currently, we limit 3 plugins for the free version.

**Comment by vothanhkiet** [comment]
Yes, I've already done that, but I feel weird that I had exactly 3 plugins, reinstall one of them and get that message. It should show when I install the fourth plugin, isn't it?
